packages/api/endpoints/token.js

```diff
@@ -212,6 +230,10 @@ const isTokenRefreshRequest = (request) =>
   request.httpMethod === 'POST'
   && request.resource.endsWith('/refresh');
 
+const isRevokeTokenRequest = (request) =>
+  request.httpMethod === 'POST'
```

Comment:
Same question as before. Does this need to be a POST? Why not a DELETE?

Reply:
See above response